sklearn/model_selection/tests/test_validation.py

```diff
@@ -211,6 +211,7 @@ def fit(self, X, Y=None, sample_weight=None, class_prior=None,
 
     def predict(self, T):
         if self.allow_nd:
+            # XXX unreached code as of v0.22
```

Comment:
Does this need a comment? The MockClassifier has presumably been copied from somewhere else, for good or bad